concurrency/WFLY-12896_EE_Concurrency_Hung_Tasks_Termination.adoc

```diff
@@ -0,0 +1,97 @@
+= EE Concurrency Hung Tasks Termination
+:author:            Eduardo Martins
+:email:             emartins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The managed executors in the EE subsystem are capable of identifying tasks which are running for a long unexpected time, also know as hung tasks, and this proposal specifies new functionality that will allows users to manually and automatically terminate such tasks.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12896[WFLY-12896]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1397[EAP7-1397]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+
+=== Other Interested Projects
+
+* NONE
+
+== Requirements
+
+* Add the following attribute to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-executor-service/[/subsystem=ee/managed-executor-service resource]:
+** *hung-task-termination-period*: the period, in milliseconds, for hung tasks automatic termination.
+
+CLI to read the attribute from the Managed Executor Service resource named *default*:
+----
+/subsystem=ee/managed-executor-service=default:read-attribute(name=hung-task-termination-period)
+----
+
+* Add the following operation to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-executor-service/[/subsystem=ee/managed-executor-service resource]:
+** *terminate-hung-tasks*: manually terminates all executor tasks that are hung.
+
+CLI to execute the operation from the Managed Executor Service resource named *default*:
+----
+/subsystem=ee/managed-executor-service=default:terminate-hung-tasks()
+----
+
+* Add the following attribute to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-scheduled-executor-service/[/subsystem=ee/managed-scheduled-executor-service resource]:
+** *hung-task-termination-period*: the period, in milliseconds, for hung tasks automatic termination.
+
+CLI to read the attribute from the Managed Scheduled Executor Service resource named *default*:
+----
+/subsystem=ee/managed-scheduled-executor-service=default:read-attribute(name=hung-task-termination-period)
+----
+
+* Add the following operation to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-scheduled-executor-service/[/subsystem=ee/managed-scheduled-executor-service resource]:
+** *terminate-hung-tasks()*: manually terminates all executor tasks that are hung.
+
+CLI to execute the operation from the Managed Scheduled Executor Service resource named *default*:
+----
+/subsystem=ee/managed-scheduled-executor-service=default:terminate-hung-tasks()
+----
```

Comment:
I think this should be moved below to the "Hard Requirements" section.

Reply:
fixed